OneSignalExample/Assets/OneSignal/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - Sending VSAttribution data from the editor
 - iOS notifications clicked event firing if the app was cold started from clicking a notification
+- ClassNotFoundException: com.onesignal.OneSignal for Android builds with minify enabled
```

Comment:
Can we add a sub note to this that you must run the export Android step again?, as I see `ExportAndroidResourcesStep` has changes.
* I see we have in our 3.x.x migration guide to run this, but this is something customers on 5.0.0 to 5.0.2 will need to be aware of they need to re-run.